modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -1,249 +1,368 @@
++
+From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
++
+* For {slesesa} 7:
++
+From the {webui}, add [systemitem]``RHEL7 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES7 x86_64``.
++
+From the command prompt, add [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64``.
+.  Synchronize the {productname} Server with the {SCC}.
+You can do this using the {webui}, or by running [command]``mgr-sync`` at the command prompt.
+. Add the new channel to your activation key.
+
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+
+// spacewalk-common-channels can't be used because centosX-uyuni-client requires centos7 channel as well, which a RHEL user would not need.
+
+.Procedure: Adding Client Tools Channels
+
+. On the {productname} Server {webui}, navigate to menu:Software[Manage > Repositories].
+. Click btn:[Create Repository] and set these parameters for the entitlement certificate:
+* In the [guimenu]``Repository Label`` field, type [systemitem]``centos7-uyuni-client``.
+* In the [guimenu]``Repository URL`` field, type the URL of the repository to mirror.
+For example, [systemitem]``https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/CentOS7-Uyuni-Client-Tools/CentOS_7/``.
+* In the [guimenu]``Has Signed Metadata?`` field, uncheck all Red Hat Enterprise Repositories.
+* Leave all other fields as the default values.
+. Click btn:[Create Repository].
+. Navigate to menu:Software[Manage > Channels].
+. Click btn:[Create Channel] and set these parameters.
+Ensure you use the correct {rhela} version:
+* In the [guimenu]``Channel Name`` field, type [systemitem]``Uyuni Client Tools for CentOS 7 (x86_64)``.
+* In the [guimenu]``Channel Label`` field, type [systemitem]``centos7-uyuni-client-x86_64``.
+* In the [guimenu]``Parent Channel`` field, select [systemitem]``rhel7-x86_64-server``.
+* In the [guimenu]``Architecture`` field, select [systemitem]``x86_64``.
+* In the [guimenu]``Repository Checksum Type`` field, select [systemitem]``sha1``.
+* In the [guimenu]``Channel Summary`` field, type [systemitem]``Uyuni Client Tools for CentOS 7 (x86_64)``.
+* In the [guimenu]``Organization Sharing`` field, select [systemitem]``Public``.
+. Click btn:[Create Channel].
+. Navigate to the [guimenu]``Repositories`` tab, check the [systemitem]``centos7-uyuni-client`` repository, and click btn:[Update repositories].
+. OPTIONAL: Navigate to the [guimenu]``Sync`` tab to set a recurring schedule for synchronization of this repository.
+. Click btn:[Sync Now] to begin synchronization immediately.
+. Add the new channel to your activation key.
 
+endif::[]
 
-The following procedure will guide you through registering a Salt client using the bootstrap script.
+== Optional: Disable {rhel} subscription-manager yum plugins
```

Comment:
What is this supposed to be attached to?

Reply:
What do you mean with "attached"?

Reply:
it says "optional", but it's a heading. It seems like it should be part of a procedure?

Reply:
Well, it's an optional section with procedures inside.

You could call it a procedure with subprocedures, I guess. But I is part of the main heading "Registering {rhel} Clients", and that's why it's a second level heading.

If it helps, you can change it to:
```
== Disable {rhel} subscription-manager yum plugins (optional)
```

Reply:
You're putting a lot of faith in markup there, Julio! :wink: